src/HPWH.hh

```diff
@@ -282,7 +282,10 @@ class HPWH : public Courier::Sender
 
         MODELS_AquaThermAire = 400, // heat exchanger model
 
-        MODELS_GenericUEF217 = 410
+        MODELS_GenericUEF217 = 410,
```

Comment:
This should probably be added to the other generic models (e.g., number 179).

Reply:
Done.